src/sentry/api/endpoints/organization_dashboards.py

```diff
@@ -57,6 +57,7 @@ def post(self, request, organization):
                     data=result.get('data'),
                 )
         except IntegrityError as e:
-            return Response(e.message, status=208)
+            # TODO(lb): hmmm this probably shouldn't be e.message?
```

Comment:
Integrity error will almost always mean that a dashboard already exists with this title, could return something like that?